platform/wayland/cog-platform-wl.c

```diff
@@ -85,6 +85,51 @@ struct _CogWlPlatformClass {
 struct _CogWlPlatform {
     CogPlatform parent;
 };
+static void resize_to_largest_output();
+static void resize_window();
+static void configure_surface_geometry();
+static void xdg_surface_on_configure(void *data, struct xdg_surface *surface, uint32_t serial);
```

Comment:
Just for ease of reading, it'd be nice to put an empty line between the forward declarations above and the shell function definitions here.